db/migrate/20240808083848_add_live_feedback_code_and_comments.rb

```diff
@@ -0,0 +1,23 @@
+class AddLiveFeedbackCodeAndComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :course_assessment_live_feedbacks do |t|
+      t.references :assessment, null: false, index: true, foreign_key: { to_table: :course_assessments }
+      t.references :question, null: false, index: true, foreign_key: { to_table: :course_assessment_questions }
+      t.references :creator, null: false, index: true, foreign_key: { to_table: :users }
+      t.datetime :created_at, null: false
+      t.string :feedback_id
+    end
+
+    create_table :course_assessment_live_feedback_code do |t|
```

Comment:
I leaned this way for naming because the natural language plural form for 'code' is still 'code', and since ruby's implied namings all follow natural language, I decided this table should be named as such